### PR TITLE
mm lod fixes

### PR DESCRIPTION
**Pull request by MadMaxFAF**
This branch is for bugfixes, general improvements, and new features. If your change is designed to alter balance, please make 
sure your changes are rebased onto [development/balance] (https://github.com/FAForever/fa/tree/development/balance), and target that branch with your PR.

### CODE REVIEW

**Comment by Garanas** [comment]
Was this an experimental branch or is there more coming?

**Comment by MadMaxFAF** [comment]
there is more to come

**Comment by Garanas** [comment]
if I am not mistaken then this branch is ready for review?

**Comment by MadMaxFAF** [comment]
i was gonna push the t3 arty here to

**Comment by Garanas** [comment]
Once it is done, click the `Ready for review` button 😄

**Comment by MadMaxFAF** [comment]
units not showing hit effects correctly 
uab2302
xab2307
ueb2302
ueb2401
xsb2302
xab2307

**Comment by Garanas** [comment]
From what perspective? That the LOD is not correct?

**Comment by MadMaxFAF** [comment]
yes the emitters are set higher that what they are displaying

File: effects/emitters/Firebig.bp [hunk 1]
```diff
@@ -10,7 +10,7 @@ EmitterBlueprint {
     AlignRotation = false,
     AlignToBone = false,
     Flat = false,
-    LODCutoff = 1000.00,
+    LODCutoff = -1.00,
```
Comment (Garanas) [inline effects/emitters/Firebig.bp:13]: something went wrong here I think

File: effects/emitters/antimatter_hit_13_emit.bp [hunk 2]
```diff
@@ -11,7 +11,7 @@ EmitterBlueprint {
     AlignRotation = true,
     AlignToBone = false,
     Flat = true,
-    LODCutoff = 500.00,
+    LODCutoff = -1.00,
```
Comment (Garanas) [inline effects/emitters/antimatter_hit_13_emit.bp:14]: why are these set to -1, but some others of the same emitter (antimatter) are still 500?

File: effects/emitters/quark_bomb_01_emit.bp [hunk 3]
```diff
@@ -10,7 +13,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = false,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/quark_bomb_01_emit.bp:16]: This is too low for the size of the effect, I think 200 - 300 is better.

File: effects/emitters/seraphim_aero_bolter_projectile_hit_01_emit.bp [hunk 4]
```diff
@@ -10,7 +12,7 @@ EmitterBlueprint {
 	AlignRotation = false,
 	AlignToBone = false,
 	Flat = true,
-	LODCutoff = -1.00,
+	LODCutoff = 100.00,
```
Comment (Garanas) [inline effects/emitters/seraphim_aero_bolter_projectile_hit_01_emit.bp:15]: I think again that 100 is quite low for the majority of the effects. Perhaps 150 here, same for 02.

**Pull request merged**